subsys/bluetooth/controller/ll_sw/ull_peripheral_iso.c

```diff
@@ -220,7 +220,10 @@ static void ticker_cb(uint32_t ticks_at_expire, uint32_t remainder,
 	/* Increment CIS event counters */
 	for (int i = 0; i < cig->lll.num_cis; i++)  {
 		cis = ll_conn_iso_stream_get(cig->cis_handles[i]);
-		cis->lll.event_count++;
+
+		if (cis->lll.handle != 0xFFFF) {
```

Comment:
A small comment on the conditional event count increment will help me (who has not completed reading the spec).